x-pack/plugins/reporting/server/usage/get_reporting_usage.ts

```diff
@@ -64,13 +69,18 @@ const getAppStatuses = (buckets: StatusByAppBucket[]) =>
   }, {});
 
 function getAggStats(aggs: AggregationResultBuckets): Partial<RangeStats> {
-  const { buckets: jobBuckets } = aggs[JOB_TYPES_KEY];
+  const { buckets: jobBuckets } = aggs[JOB_TYPES_KEY] as AggregationBuckets;
   const jobTypes = jobBuckets.reduce((accum: JobTypes, bucket) => {
-    const { key, doc_count: count, isDeprecated } = bucket;
+    const { key, doc_count: count, isDeprecated, sizeMax, sizeMin, sizeAvg } = bucket;
     const deprecatedCount = isDeprecated?.doc_count;
     const total: Omit<AvailableTotal, 'available'> = {
       total: count,
       deprecated: deprecatedCount,
+      output_size: {
+        max: sizeMax?.value ?? null,
+        min: sizeMin?.value ?? null,
+        avg: sizeAvg?.value ?? null,
+      },
```

Comment:
I wonder whether a percentile based aggregation would not provide better insight into the questions we may have regarding output size ([es docs](https://www.elastic.co/guide/en/elasticsearch//reference/current/search-aggregations-metrics-percentile-aggregation.html)).

Would it be more useful to know that the average output size is, e.g., `500kb` or that 99% of outputs are below `700kb`?

Percentile based aggregation can be more expensive to calcuate, which might disqualify it, but from what I can tell it looks pretty safe (also considering we are working with report documents so not sure there will be millions).

Something like:

```ts
output_size: {
  25th: ...,
  50th: ...,
  75th: ...,
  95th: ...,
  99th: ..., // outliers
}
```

Let me know what you think @tsullivan !

Reply:
++ I am in favor of this. At the time I implemented it with `avg`, I was hoping to use median instead, but there isn't a median aggregation. With percentiles, we get the median from the 50th percentile. This is good!